TbsCore/Helpers/HttpHelper.cs

```diff
@@ -36,6 +37,7 @@ public static string SendPostReq(Account acc, RestRequest req)
 
         public static HtmlAgilityPack.HtmlDocument SendGetReq(Account acc, string url)
         {
+            acc.Wb.RestClient.BaseUrl = new Uri($"{acc.AccInfo.ServerUrl}");

```

Comment:
wouldn't it be easier/cleaner to just use `new Uri(acc.AccInfo.ServerUrl);`?

Reply:
lol how i forgot this =))